Prolific/Prolific/Utils/Navigation/NavigationManager.m

```diff
@@ -39,9 +39,10 @@ + (void)presentRegistrationScreenWithNavigationController:(UINavigationControlle
     [navController pushViewController:registerViewController animated:YES];
 }
 
-+ (void)presentProjectDetailsViewControllerForProject:(Project *)project navigationController:(UINavigationController *)navController {
++ (void)presentProjectDetailsViewControllerForProject:(Project *)project forUser:(User *)user navigationController:(UINavigationController *)navController {
     ProjectDetailsViewController *const projectDetailsViewController = [[ProjectDetailsViewController alloc] init];
```

Comment:
note: if you have access to the initializer, it's cleaner to write a new initializer that takes in these properties versus having mutable properties on the vc